### ISSUE TITLE
Uncaught exception - alloc.c(177) : max_string_size reached

### ISSUE THREAD

**Issue by mynameGY** [HaxeFoundation/neko#189]
How to deal with it please help me

**Comment by ppenzin** [comment]
Hi @mynameGY , can you please add more details. What neko code are you running, can you post the source?

**Comment by mynameGY** [comment]
I used haxeflixel to build flash and failed.
When I delete some of the resources under my assets, I can build again successfully.

### Here's the details error ### ：
Called from ? line 1
Called from CommandLineTools.hx line 1385
Called from CommandLineTools.hx line 25
Called from CommandLineTools.hx line 126
Called from CommandLineTools.hx line 604
Called from lime/project/PlatformTarget.hx line 70
Called from lime/tools/platforms/FlashPlatform.hx line 260
Called from lime/tools/helpers/FlashHelper.hx line 872
Called from lime/tools/helpers/FlashHelper.hx line 685
Called from format/swf/Writer.hx line 60
Called from format/swf/Writer.hx line 1425
Called from format/tools/Deflate.hx line 33
Called from /usr/lib/haxe/std/neko/_std/haxe/zip/Compress.hx line 48
Called from /usr/lib/haxe/std/haxe/io/Bytes.hx line 457
Uncaught exception - alloc.c(177) : max_string_size reached
Build halted with errors (haxelib.exe).

**Comment by ppenzin** [comment]
Looks like the string describing resources becomes too long to be allocated in Neko. You would probably have to report the issue for the tool that is getting this error.

@ncannasse, @andyli, what do yo think?

**Comment by ncannasse** [comment]
This is some issue related to lime and should be handled at lime level. It seems weird that there is a 256MB limit to resources as I think some projects have more than that.

**Comment by ppenzin** [comment]
This needs to be fixed in Lime ([openfl/lime][l], right @ncannasse?). To get help with fixing it you should report this issue in the lime repository.

[l]: https://github.com/openfl/lime

**Comment by mynameGY** [comment]
Okay, thank you very much @ppenzin .I really look forward to ncannasse's help. Thank @ncannasse in advance.

**Comment by ncannasse** [comment]
@ppenzin yes, this is somewhat lime specific. Maybe recompiling and running the build tool with HashLink would prevent this error from occurring.

**Comment by ppenzin** [comment]
Are strings allocated differently in HashLink? I haven't gotten to building it yet since I got stuck with FreeBSD build of Haxe :)

**Comment by ncannasse** [comment]
Well you don't have the 256MB limit.
